### PR TITLE
Validating passwords on registration and settings issue #75

### PR DESCRIPTION
**Pull request by victorlian**
This pull requests finishes issue #75 

1. Upon registration, password and confirm password mismatch will be detected.
image

2. Upon changing password, password and confirm password mismatch will be detected.
image

### CODE REVIEW

File: src/main/webapp/app/account/index.ts [hunk 1]
```diff
@@ -18,4 +19,4 @@ export * from './settings/settings.component';
 export * from './settings/settings.route';
 export * from './social/social-register.component';
 export * from './social/social.route';
-export * from './account.route';
+export * from './account.route';
```
Comment (st970703) [inline src/main/webapp/app/account/index.ts]: Hey,
What has changed between Line 21 and Line 22?

File: src/main/webapp/app/account/register/register.component.html [hunk 2]
```diff
@@ -93,20 +93,21 @@ <h1>Registration</h1>
                 <div class="form-group">
                     <label class="form-control-label" for="confirmPassword">New password confirmation</label>
                     <input type="password" class="form-control" id="confirmPassword" name="confirmPassword" #confirmPasswordInput="ngModel" placeholder="Confirm the new password"
-                            [(ngModel)]="confirmPassword" minlength=4 maxlength=50 required>
+                            [(ngModel)]="confirmPassword" minlength=4 maxlength=50 required validateEqual="password">
                     <div *ngIf="confirmPasswordInput.dirty && confirmPasswordInput.invalid">
-                        <small class="form-text text-danger" *ngIf="confirmPasswordInput.errors.required"
-                              >
+                        <small class="form-text text-danger" *ngIf="confirmPasswordInput.errors.required">
```
Comment (st970703) [inline src/main/webapp/app/account/register/register.component.html]: Hey,
Are changes like between line 98 just formatting changes?

File: src/main/webapp/app/account/index.ts (hunk 1 above)
Comment (victorlian) [reply src/main/webapp/app/account/index.ts]: apparently a new line character is removed... shouldn't affect anything :)

File: src/main/webapp/app/account/register/register.component.html (hunk 2 above)
Comment (victorlian) [reply src/main/webapp/app/account/register/register.component.html]: Yea some of them were just formatting changes to make the source code more consistent.

**Comment by st970703** [comment]
Hey,
Looks good to me.

**Comment by victorlian** [comment]
@softeng-701 has been reviewed, ready to be merged. Thanks.

**Comment by victorlian** [comment]
The commit above is has squashed all the commits.

**Pull request merged**